qa/L0_stability_perf/check_results.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import glob
+import os
+from collections import defaultdict
+import re
+
+
+class TestOutputValidator:
+    """
+    Functions that validate the output
+    of the test
+    """
+    def __init__(self, test_name, threshold):
+        self._tolerance_percent = threshold
+
+        check_function = self.__getattribute__(f'check_{test_name}')
+
+        if check_function():
+            sys.exit(0)
+        else:
+            sys.exit(1)
+
+    def check_perf_stability(self):
+        """
+        Get the perf results from each model analyzer run
+        as well as perf_analyzer run and compare them
+        """
+
+        log_paths = glob.glob(os.path.join(os.getcwd(), f"*.log"))
+
+        # Split the logs up into log pairs model_name: [model_name.test.log, model_name.perf.log]
```

Comment:
Could you please explain what's the difference between `model_name.test.log` and `model_name.perf.log`?

Reply:
.test.log is the model-analyzer log, same as all the other tests. .perf.log is the perf analyzer log.

Reply:
Would be great to add a comment but I was just wondering which files they refer to.